enterprise_access/apps/bffs/serializers.py

```diff
@@ -152,12 +152,27 @@ class EnterpriseCustomerSerializer(BaseBffSerializer):
     show_integration_warning = serializers.BooleanField()
 
 
+class EnterpriseCustomerUsersSerializer(BaseBffSerializer):
```

Comment:
This represent a _single_ `EnterpriseCustomerUser` record, yeah? If so, should this be named as `EnterpriseCustomerUserSerializer`?